docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -516,59 +521,59 @@ Timeout before killing an unresponsive chain. Defaults to `5s`.
 
 #### `--creation-tx-fee` (int):
 
-Transaction fee, in nAVAX, for transactions that create new state. Defaults to `1000000` nAVAX (.001 AVAX) per transaction.
+Transaction fee, in nAVAX, for transactions that create new state. Defaults to `1000000` nAVAX (.001 AVAX) per transaction. This can only be changed in a local network.
```

Comment:
```suggestion
Transaction fee, in nAVAX, for transactions that create new state. Defaults to `1000000` nAVAX (.001 AVAX) per transaction. This can only be changed on a local network.
```
just to align it with others